frontend/components/Layout.tsx

```diff
@@ -1,14 +1,25 @@
-import Header from './Header';
-import Footer from './Footer';
+
 import { useEffect, useLayoutEffect } from 'react';
+import {motion} from 'framer-motion';
 
 const Layout = ({ children }: any) => {
+  const variants = {
+    hidden: { opacity: 0, x: 0, y: -200 },
```

Comment:
I think making the y value a bit smaller would lead to a subtle, better looking animation, especially in the blog route where the the blog post currently flies through the header.
What are your opinions on this?

Reply:
We'll fix that soon, but later on we'll look into another navbar design, with background. What do you say about that?

Reply:
That is fine for a future PR but for now, as the code is going into production, I suggest making the y values smaller